qiskit/providers/aer/openpulse/qobj/digest.py

```diff
@@ -61,6 +62,15 @@ def digest_pulse_obj(qobj_input, backend_options, noise_model):
         raise ValueError('Qobj must have hamiltonian in config to simulate.')
     hamiltonian = config_dict_sim['hamiltonian']
 
+    # Warnings for untested features
```

Comment:
Can you move all this code to a function? The `digest_pulse_obj` is already huge enough:
Maybe something like:
```
warnings(hamiltonian, noise_model, qobj['experiments'])
```
And do the check/`warn()` there.

Reply:
Makes sense, thanks! I've made this change.